README.md

```diff
@@ -57,41 +60,67 @@ To get a local copy up and running follow these steps.
       -e POSTGRES_HOST_AUTH_METHOD=trust \
       ankane/pgvector
     ```
+    If you already have an instance of `pgvector` you can create a database for Dewy and configure Dewy use it using the `DB` env var (see below).
+    
 1. Install Dewy
     ```
     pip install dewy
     ```
-1. Fire up Dewy
-    ```sh
-    // Configure your OpenAI key (optional - local models will be used if not provided)
-    export OPENAI_API_KEY=...
 
-    // Configure your pgvector endpoint (this assumes you installed with Docker as shown previously)
-    export DB=postgresql://dewydbuser:dewydbpwd@localhost/dewydb
+    This will install Dewy in your local Python environment.
+
+1. Configure Dewy.
+    Dewy will read env vars from an `.env` file if provided. You can also set these directly
+    in the environment, for example when configuring an instance running in docker / kubernetes.
```

Comment:
Using the `.env` config method, since this is running locally. Side note - do we want to make this something like `~/.dewyrc` so the path isn't dependent on where you run the script?